config/default.js

```diff
@@ -61,11 +61,11 @@ module.exports = {
     search: {
       enabled: true,
       indexName: 'docs',
-      algoliaAppId: null,
-      algoliaSearchKey: null,
-      algoliaAdminKey: null,
-      excerptSize: 100000,
-      engine: 'localsearch',
+      algoliaAppId: process.env.ALGOLIA_APP_ID,
```

Comment:
What your configure in GitHub Actions workflow script is correct.
However, I don't understand how our project builds indexes.

Will indexes get built when calling `yarn build` in package.json which is called by GitHub Actions
https://github.com/enriikke/gatsby-gh-pages-action/blob/main/index.ts#L44?
Do we need extra command for building indexest?

Reply:
It will be done automatically by `gatsby-plugin-algolia`, when run `yarn build`, it'll build index.

Reply:
Thanks.